clang/tools/cconv-standalone/CConvStandalone.cpp

```diff
@@ -132,6 +131,18 @@ int main(int argc, const char **argv) {
   CcOptions.AddCheckedRegions = OptAddCheckedRegions;
   CcOptions.EnableAllTypes = OptAllTypes;
   CcOptions.DisableCCTypeChecker = OptDiableCCTypeChecker;
+  //Add user specified function allocators
+  std::string Malloc = OptMalloc.getValue();
+  std::string delimiter = ",";
+  size_t pos = 0;
+  std::string token;
+  while ((pos = Malloc.find(delimiter)) != std::string::npos) {
+    token = Malloc.substr(0, pos);
+    CcOptions.FunctionAllocs.insert(token);
+    Malloc.erase(0, pos + delimiter.length());
+  }
+  token = Malloc;
```

Comment:
This is flaky. Are you sure there is no way, we can pass list of the options? I mean is it possible to have `cl::opt<std::vector<string>>` that way you don't need to do this.

Reply:
Yes, there is a way to use a vector here and get rid of this `while` loop and instead replace it with a `split` function found in boost that is super simple. I used a set because in `Utils.cpp` and `ConstraintResolver.cpp`, I have to check whether `FuncName`s are inside of this structure and sets have `O(log n)` search and vectors have `O(n)` search. There are only a few elements in it probably, so I'll just use a vector.